submission/distributor_test.go

```diff
@@ -100,43 +100,43 @@ var (
 )
 
 // newNoLogClient is LogClientBuilder that always fails.
-func newNoLogClient(_ *loglist2.Log) (client.AddLogClient, error) {
+func newNoLogClient(_ *loglist3.Log) (client.AddLogClient, error) {
 	return nil, errors.New("bad log-client builder")
 }
 
-func sampleLogList() *loglist2.LogList {
-	var ll loglist2.LogList
+func sampleLogList() *loglist3.LogList {
+	var ll loglist3.LogList
 	if err := json.Unmarshal([]byte(testdata.SampleLogList2), &ll); err != nil {
```

Comment:
The `testdata.SampleLogList3` should be used to unmarshal to `*loglist3.LogList` although the loglist is backward compatible.

Reply:
To be addressed in #966.